### PR TITLE
FS-1015 Add logging util

### PR DESCRIPTION
**Pull request by harryyo**
Logging util from authenticator moved to utils repo

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,22 +1,28 @@
...
+    "version": "0.2.0",
     "description": "Utils for the fsd-tech team",
     "long_description": None,
-    "author": "DLUHC",
+    "author": "Version 1",
     "author_email": None,
     "maintainer": None,
     "maintainer_email": None,
-    "url": "https://github.com/communitiesuk/funding-service-design-utils",
+    "url": None,
```
Comment (gidsg) [inline setup.py:22]: can we revert these changes?

File: setup.py [hunk 2]
```diff
@@ -1,22 +1,28 @@
 # -*- coding: utf-8 -*-
 from setuptools import setup
 
-packages = ["fsd_utils"]
+packages = ["fsd_tech"]
 
 package_data = {"": ["*"]}
```
Comment (gidsg) [inline setup.py:6]: Should this just include the utils folder?

File: setup.py (hunk 1 above)
Comment (harryyo) [reply setup.py:22]: Ah yes sorry - unintentional copies

**Review by SentientHousePlant — APPROVED** [review]
LGTM! <3
